.env.dist

```diff
@@ -14,6 +14,8 @@ APP_SECRET=92d6f6713081a061210b56723f0389e5
 # Delivery is disabled by default via "null://localhost"
 MAILER_URL=null://localhost
 ###< symfony/swiftmailer-bundle ###
+MAILER_FROM_EMAIL=tools.svgtranslate@tools.wmflabs.org
+MAILER_TO_EMAIL=
```

Comment:
`.env` is the new standard so this is probably fine, but I think this means it won't work in Event Metrics right now :(